tizen/src/sharedpreference_helper.h

```diff
@@ -0,0 +1,8 @@
+#ifndef FLUTTER_PLUGIN_WORKMANAGER_SHAREDPREFERENCE_HELPER_H_
+#define FLUTTER_PLUGIN_WORKMANAGER_SHAREDPREFERENCE_HELPER_H_
+
+#include <app_preference.h>
+
+const char* DISPATHCER_HANDLE_KEY = "WMANAGER_TIZEN_DISPATCHER_HANDLE_KEY";
```

Comment:
Is there a reason to separate the code into a separate header file?

Reply:
At first, I intended to put shared preference related codes in one class. But, I don't need it right now, so I moved it to `constants.h`.